### PR TITLE
Feature/feature 183 ubim upate

### PR DESCRIPTION
**Pull request by Prabhu-Dell**
# Description
Updated UBI image

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| dell/csm#183 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Tested in Kubernetes cluster by building a fresh image

### CODE REVIEW

File: docker-files/Dockerfile.ubi.min [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Dockerfile to build PowerStore CSI Driver
 # based on UBI-minimal image
 # Requires: RHEL host with subscription
-FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:3aa3f379a81013bd3264faa0af87d201cdaa5981050d78c567b48fdfd5b38bb8
+FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:21504085e8d740e62b52573fe9a1a0d58a3e7dba589cac69734ad2fa81d66635
```
Comment (bpjain2004) [inline docker-files/Dockerfile.ubi.min:4]: Please put a comment describing the image for which this sha key corresponds to. For ex for this:
# ubi8/ubi-minimal:8.5-230.1645809059

**Review by spriya-m — COMMENTED** [review]
@Prabhu-Dell , can you please add the tests done - docker build passing, installation, etc. ?

**Comment by prablr79** [comment]
@Prabhu-Dell we are going code freeze can you quickly check on the comments and update this PR asap?

File: docker-files/Dockerfile.ubi.min (hunk 1 above)
Comment (harishp8889) [reply docker-files/Dockerfile.ubi.min:5]: done

**Review by spriya-m — APPROVED** [review]

**Review by harishp8889 — APPROVED** [review]

**Pull request merged**